mox/src/lib.rs

```diff
@@ -347,8 +342,9 @@ impl ToTokens for MoxAttr {
     }
 }
 
-struct MoxExpr {
-    expr: syn::Expr,
+enum MoxExpr {
```

Comment:
I'm not sure I follow what this is for -- is it only to add `#[allow(unused_braces)]` on some blocks?

Reply:
Exactly. Let me know if you think it warrants a comment or you'd rather just blanket `#[allow(unused_braces)]`.

Reply:
Yeah, let's just slap the unused braces lint on both options if we need to.

Reply:
Fixed